test/specs/instance.spec.ts

```diff
@@ -1014,10 +1014,10 @@ describe('instance', () => {
   });
 
   describe('complete', () => {
-
     it('gets triggered if an error occurs', () =>
+
       User
-        .findOne({where: ['asdasdasd']})
+        .findOne<User>({where: <any>{ bad: 'asdasdasd' }})
```

Comment:
Incidentally, the newer Sequelize typings actually turn bad `where` clauses into a compiler error, making this sort of error a bit harder to trigger.